### PR TITLE
update test cases

### PR DESCRIPTION
**Pull request by NicoYuan1986**
1. update some error code account of milvus update #23075
2. add test case that flush does not work #26820

### CODE REVIEW

File: tests/python_client/testcases/test_utility.py [hunk 1]
```diff
@@ -241,7 +241,7 @@ def test_loading_progress_not_existed_collection_name(self):
         self.collection_wrap.construct_from_dataframe(c_name, df, primary_field=ct.default_int64_field_name)
         self.collection_wrap.create_index(ct.default_float_vec_field_name, index_params=ct.default_flat_index)
         self.collection_wrap.load()
-        error = {ct.err_code: 1, ct.err_msg: "describe collection failed: can't find collection"}
+        error = {ct.err_code: 4, ct.err_msg: "collection default:not_existed_name: collection not found"}
```
Comment (binbinlv) [inline tests/python_client/testcases/test_utility.py:244]: what does this mean? collection name?

File: tests/python_client/testcases/test_utility.py [hunk 2]
```diff
@@ -285,7 +285,7 @@ def test_wait_for_loading_collection_not_existed(self):
         self.utility_wrap.wait_for_loading_complete(
             c_name,
             check_task=CheckTasks.err_res,
-            check_items={ct.err_code: 1, ct.err_msg: "can't find collection"})
+            check_items={ct.err_code: 4, ct.err_msg: "collection default:utility_muVBWmM8: collection not found"})
```
Comment (binbinlv) [inline tests/python_client/testcases/test_utility.py:288]: what does this mean? collection name?

File: tests/python_client/testcases/test_utility.py [hunk 3]
```diff
@@ -601,9 +601,9 @@ def test_rename_collection_not_existed_collection(self):
         new_collection_name = cf.gen_unique_str(prefix)
         self.utility_wrap.rename_collection(old_collection_name, new_collection_name,
                                             check_task=CheckTasks.err_res,
-                                            check_items={"err_code": 1,
-                                                         "err_msg": "can't find collection: {}".format(
-                                                             collection_w.name)})
+                                            check_items={"err_code": 4,
+                                                         "err_msg": "collection 1:test_collection_non_exist: "
```
Comment (binbinlv) [inline tests/python_client/testcases/test_utility.py:605]: what does this mean?

Comment (NicoYuan1986) [reply tests/python_client/testcases/test_utility.py:605]: > what does this mean?

collection 1 means the first param of rename_collection, 
test_collection_non_exist is old_collection_name.

File: tests/python_client/testcases/test_utility.py (hunk 1 above)
Comment (NicoYuan1986) [reply tests/python_client/testcases/test_utility.py:244]: Yes. In this case, `loading_progress("not_existed_name", check_task=CheckTasks.err_res, check_items=error)` the collection name "not_existed_name" is given.

File: tests/python_client/testcases/test_utility.py (hunk 2 above)
Comment (NicoYuan1986) [reply tests/python_client/testcases/test_utility.py]: Yes. I have modified the code here.

Comment (binbinlv) [reply tests/python_client/testcases/test_utility.py]: Maybe the message default:collectionName more user friendly?  Please track this.

**Pull request merged**